resim/metrics/default-report-metrics/entrypoint.py

```diff
@@ -0,0 +1,25 @@
+#!/bin/python3
+
+import asyncio
+import pathlib
+import sys
+
+from resim.metrics.default_report_metrics import main as default_report_metrics_main
+
+REPORT_METRICS_CONFIG_PATH = "/tmp/resim/inputs/report_config.json"
+
+
+async def main() -> None:
+    report_metrics_config = pathlib.Path(REPORT_METRICS_CONFIG_PATH)
+    is_report_metrics_mode = report_metrics_config.is_file()
+    if is_report_metrics_mode:
+        # Call the default_report_metrics main function:
+        await default_report_metrics_main()
+    else:
+        # Otherwise, we can't do anything: exit 1
+        print("Metrics Build only compatible with reports")
+        sys.exit(1)
```

Comment:
[nit] I lean towards just doing this in the default report metrics file, but I won't block on it. Eventually, I hope that file can look much more like this one once we've abstracted out and tested most of it.

Reply:
SGTM